prog/dftb+/lib_io/formatout.F90

```diff
@@ -146,14 +146,13 @@ subroutine writeGenFormat_fid(fd, coord, species, speciesName, latVec, origin, t
     logical, intent(in), optional :: tFracCoord
 
     integer :: nAtom, nSpecies
-    character(6) :: formatSpecies
+    character(mc) :: formatSpecies, formatCoordinates
     integer :: ii
     logical :: tFractional, tHelical, tPeriodic
     real(dp) :: invLatVec(3,3)
 
-100 format(I5," ",A2)
-101 format("(",I2.2,"A3)")
-102 format(I5,I2,3E20.10)
+101 format(1X,I0," ",A2)
```

Comment:
Is there a specific need for the leading space? Also I would prefer to use either `1X` or `" "` for spaces to keep the format statements consistent.

Reply:
Not really, resulting file just looks neater to me. The above '(a)' discussion replicates this behaviour anyway.